content/applications/hr/time_off.rst

```diff
@@ -0,0 +1,625 @@
+approved time off, the planned time off, and the total ending balance with those numbers factored
+in. The time off balance labeled as :guilabel:`Left` is what is displayed next to the icon in the
+summary. Click the :guilabel:`? (question mark)` again to close the pop-up.
+
+The legend on the right side of the calendar view displays the various time off types, with their
+corresponding colors. The status of the time off requests are shown as well. Time off that has been
+validated appears in a solid color (in the color specified in the Time Off Types part of the
+legend). Time off requests that still needs to be approved appear with white stripes in the color.
+Refused time off requests have a colored line through the dates.
+
+New time off requests can be made from the dashboard. Click the :guilabel:`New Time Off` button at
+the top of the dashboard, and a new :ref:`time off form <time_off/request-time-off>` appears. New
+allocation requests can also be made from the dashboard. Click the :guilabel:`Allocation Request`
+button at the top of the dashboard to request more time off, and a new :ref:`allocation form
+<time_off/request-allocation>` appears form appears.
+
+.. image:: time_off/dashboard.png
+   :align: center
+   :alt: Time off dashboard view with the legend, time off summaries, and view buttons highlighted.
+
+My time off
+-----------
+
+To view a list of all the users time off requests, navigate to :menuselection:`Time Off application
+--> My Time Off --> My Time Off`. All time off requests appear in a list view, both past and
+present. Other than the employee's name, the list includes the following information for each
+request: the :guilabel:`Time Off Type`, :guilabel:`Description`, :guilabel:`Start Date`,
+:guilabel:`End Date`, :guilabel:`Duration`, and the :guilabel:`Status`. A new time off request can
+be made from this view. Click the :guilabel:`Create` button to :ref:`request time off
+<time_off/request-time-off>`.
+
+My allocations
+--------------
+
+To view a list of all the users allocations, navigate to :menuselection:`Time Off application -->
+My Time Off --> My Allocations`. All allocations and requested allocations appear in a list view.
+The information presented includes: the :guilabel:`Time Off Type`, :guilabel:`Description`,
+:guilabel:`Duration`, :guilabel:`Allocation Type`, and the :guilabel:`Status`. A new allocation
+request can be made from this view. Click the :guilabel:`Create` button to :ref:`request an
+allocation <time_off/request-allocation>`.
+
+Overview
+========
+
+To view a color-coded schedule of both the user's time off, and/or the team managed by them,
+navigate to :menuselection:`Time Off application --> Overview`. This presents a calendar with the
+default filter of :guilabel:`My Team`, in a month view. To change the time period displayed, click
+on either the :guilabel:`Day`, :guilabel:`Week`, or :guilabel:`Year` buttons to present the calendar
```

Comment:
also month

Reply:
Since month is the default and is already selected, I didn't include it in the list, but I can definitely add it.